pyfr/solvers/aceuler/kernels/rsolvers/roe.mako

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+<%include file='pyfr.solvers.aceuler.kernels.flux1d'/>
+
+<% eps = 0.001 %>
+<% zeta = c['ac-zeta'] %>
+<% rzeta = 1./zeta %>
+
+<%pyfr:macro name='rsolve_t1d' params='ul, ur, nf'>
+    // Compute the left and right fluxes + velocities and pressures
+    fpdtype_t fl[${nvars}], fr[${nvars}];
+    fpdtype_t va[${ndims}], dv[${ndims}];
+    fpdtype_t v1[${nvars}], v2[${nvars}], v3[${nvars}];
```

Comment:
I think we can eliminate these arrays by simply consuming the values (which go into nf after being processed) as soon as we generate them.